src/modules/service-catalog/translations/en-us.yml

```diff
@@ -40,3 +40,13 @@ parts:
       title: "Empty state go to homepage link"
       screenshot: "https://drive.google.com/file/d/1D_3VXzWSuj9f34PsFPa0b4uELz8-T9ll/view?usp=drive_link"
       value: "Go to the homepage"    
+  - translation:
+      key: "service-catalog.item.read-more"
+      title: "Button to show whole description of the service catalog item"
+      screenshot: "https://drive.google.com/file/d/1pXglNRRqa23LoA7CuZOiAKQAuwkRKFqN/view?usp=drive_link"
```

Comment:
Seems like there's an issue with the screenshot 😄
<img width="1792" alt="image" src="https://github.com/user-attachments/assets/d5d4f346-6d6c-437c-9911-2ef6a1e7f3c6">

Reply:
Thanks, I fixed the screenshots.